components/ExplorerDownloadDialog.js

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Grid,
+  Tab,
+} from "@material-ui/core";
+
+const ExplorerDownloadDialog = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleClick = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Box>
+      <Tab label="Download" onClick={handleClick} />
+      <Dialog open={open} onClose={handleClose}>
+        <DialogTitle>Download Database</DialogTitle>
+        <DialogContent dividers>
+          <DialogContentText>
+            The database of the Sustainable Development Report 2021 contains the
+            SDG Index rankings, overall scores, spillover scores, dashboards and
+            trends for all 17 SDGs and all 100+ indicators. The Codebook sheet
+            lists indicator metadata and sources.
+          </DialogContentText>
+        </DialogContent>
+        <DialogContent>
+          <Box paddingBottom={2}>
+            <Grid container spacing={2}>
+              <Grid item xs={12} lg={6}>
+                <Button
+                  component="a"
+                  href="TODO"
```

Comment:
```suggestion
                  href={DATABASE_DOWNLOAD_URL}
```

Can you define `DATABASE_DOWNLOAD_URL` in `config.js` with value `/TODO`? Then import in this component:

`import { DATABASE_DOWNLOAD_URL } from "root/config.js"`

(Example in `pages/chapters.js`)

---------------------

Can you also add this import in `pages/downloads.js` and replace `url: "TODO"` with `url: DOWNLOAD_DATABASE_URL` for the database download?

```js
  {
    label: "Database",
    type: "Excel",
    image: "database.png",
    url: DOWNLOAD_DATABASE_URL
  },
```